cmd/hub/cli/cli.go

```diff
@@ -0,0 +1,72 @@
+package cli
```

Comment:
`cmd/hub/cli` and `cmd/buck/cli` can be initialized from `main` functions, e.g., `cmd/hub` initialized them both, while `cmd/buck` only initializes `cmd/buck/cli`.